personal-website/src/app/navbar.js

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+
+export default function Navbar() {
```

Comment:
By convention, components should be put in a `components` folder. It should also be a .tsx file